src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,20 @@
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo/CommetInfo';
+
+type Props = {
+  comments: Comment[] | null;
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="ui comments">
+    <h3 className="ui dividing header">Comments</h3>
+    <ul className="comment-list">
+      {comments && (
```

Comment:
do as @Bohdan-Maksymiuk has suggested,
and comments couldn't be false, it's empty or not empty array